### DISCUSSION TITLE
🤝 友链申请

### DISCUSSION THREAD

**Discussion by dsrkafuu** [dsrkafuu/dsr-blog#5] [公告]
友链申请。

https://blog.dsrkafuu.net/friends/

**Comment by dsrkafuu** [comment]
## 申请条件

- 建站大于等于一年，无内容类型要求，原创内容为主，翻译转载亦可
- 确保站点可用性，中国大陆可除外，启用全站 SSL
- 我会定期访问站点，若暂时不可用将暂时移除链接并移动至列表末尾
- 不接受单向友链

## 申请途径

- 本页留言
- Twitter [@dsrkafuu](https://twitter.com/dsrkafuu)

## 申请模板

```markdown
- 名称：DSRBLOG
- 简介：Internet for people, not profit
- 链接：https://blog.dsrkafuu.net/
- 图标：https://cdn.dsrkafuu.net/avatar/dsrkafuu_256p.webp
```

**Reply by dsrkafuu** [reply]
请不要在这条评论上直接回复

**Comment by a506488043** [comment]
名称：Saiita's BLog
简称：好记性不如烂笔头
网址：https://www.saiita.com.cn
头像：https://cdn.saiita.com.cn/wp-content/uploads/2021/02/1613143399-软件.png

**Reply by dsrkafuu** [reply]
欢迎！已添加~

**Comment by qqAys** [comment]
- 名称：qqAys
- 简介：学途漫漫，朋辈同行
- 链接：https://qqays.xyz/
- 图标：https://qqays.xyz/usr/favicon.png

**Reply by dsrkafuu** [reply]
没找到你站点上的友链页/模块哦~

**Reply by qqAys** [reply]
很抱歉，现在已经添加上了！点击下方链接查看（可能需要清除浏览器缓存才能达到最佳效果）
[友链和评论 - qqAys](https://qqays.xyz/friends-and-comments.html)
祝好

**Reply by dsrkafuu** [reply]
已添加~

**Comment by LanYunDev** [comment]
已添加贵站啦~博主文章让我学到很多!❤️
名称：LanYunのBlog
简介：Share Technology.
链接：https://lanyundev.com/
图标：https://lanyundev.com/img/logo.jpg.webp

**Reply by dsrkafuu** [reply]
欢迎~ 已添加

**Comment by weekdaycare** [comment]
- 名称：星日语
- 简介：星语日点灯
- 链接：https://weekdaycare.cn
- 图标：https://weekdaycare.cn/asset/avatar.svg

大佬太强啦，基安来的，膜拜一下，我的是动态友链，佬直接写issue里就可以啦
https://github.com/weekdaycare/friends/issues

**Reply by dsrkafuu** [reply]
欢迎~ 已添加

**Comment by LanYunDev** [comment]
收到,晚些将更新

**Reply by weekdaycare** [reply]
佬好像是看错了🤣俺都加上🎉

**Comment by WincerChan** [comment]
博主的博客很好看，申请下友链～
- 名称：Wincer's Blog
- 简介：喜欢折腾新技术，却没多大耐心
- 链接：https://blog.itswincer.com/
- 图标：https://ae01.alicdn.com/kf/HTB1DdGHXizxK1RjSspj763S.pXax.png

**Reply by dsrkafuu** [reply]
欢迎~ 已添加

**Comment by a506488043** [comment]
已添加友链
我的博客地址：saiita.com.cn




程厚锋
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "dsrkafuu/dsr-blog"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年11月24日(星期四) 下午4:14
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [dsrkafuu/dsr-blog] 🤝 友链申请 (Discussion #5)





 
博主很好看，申请下友链～
  
名称：Wincer's Blog
 
简介：喜欢折腾新技术，却没多大耐心
 
链接：https://blog.itswincer.com/
 
图标：https://ae01.alicdn.com/kf/HTB1DdGHXizxK1RjSspj763S.pXax.png

**Comment by a506488043** [comment]
你的网址是什么




程厚锋
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "dsrkafuu/dsr-blog"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年11月26日(星期六) 中午11:48
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [dsrkafuu/dsr-blog] 🤝 友链申请 (Discussion #5)





 
欢迎~ 已添加

**Comment by hifocus** [comment]
你好，关注你挺久啦~ 惶心博客 [huangxin.dev](https://huangxin.dev) 申请下友链：
```text
- 名称：惶心博客
- 简介：「别说万事转头空 未转头时为梦」
- 链接：https://huangxin.dev/
- 图标：https://www.gravatar.com/avatar/054ff9853488d20f0e8a5dda18a40076?s=500 或者 https://images.hxis.top/avatar/latest.jpg
```

更多可参见博客友链陈述文章：https://huangxin.dev/friends/
希望通过~ 确认以后会及时在 https://friends.huangxin.org 添加你的博客~

**Comment by ACT-02** [comment]
博主您好！目前 PingFang.ttc 已更新到 19.0d3e2 版本，希望能够及时更新一下，并附上未修改的字体原文件，谢谢博主！
